packages/design-tokens/src/index.ts

```diff
@@ -1,9 +1,44 @@
 // These are exposed at the top level because it seems like they are the most useful and relevent to usage of design tokens.
+
+/**
+ * @deprecated
+ * - Not needed if you are using `KaizenProvider` from `@kaizen/components` or `defaultPreset` from next-services.
+ * - If you require direct usage of these tokens in js you can `import { tokens } from "@kaizen/design-tokens/js"`
+ */
 export * from "./themes"
+
+/**
+ * @deprecated
```

Comment:
Oh? Does this apply the comment to everything exported?

Reply:
I thought it did, but apparently I never tested this and it does not work haha

Reply:
Need some way to deprecate external usage...